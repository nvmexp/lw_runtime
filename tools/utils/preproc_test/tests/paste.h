#define EXPAND( a ) a
+EXPAND(+)+
EXPAND(1)2
EXPAND(a)2
EXPAND(1)a
+EXPAND(1.e+1)+
*EXPAND(1)*
.EXPAND(1).
.EXPAND(.1).
'a'EXPAND('b')'c'
EXPAND(1+)_

#define NUMBER()     1
#define UNDERSCORE() _
NUMBER()UNDERSCORE()
UNDERSCORE()NUMBER()
