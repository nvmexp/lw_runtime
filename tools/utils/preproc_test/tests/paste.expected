#line 1 "paste.h"

+ + +
1 2
a 2
1 a
+1.e+1 +
*1*
. 1 .
. .1 .
'a''b''c'
1 +_



1 _
_ 1
